src/app/collection/collection.module.ts

```diff
@@ -25,7 +25,7 @@ import { HeaderInfo502Component } from './pages/collection-502/components/header
 import { CuratorCardComponent } from './pages/collection-502/components/curators/components/curator-card/curator-card.component';
 import { TitleComponent } from './pages/collection-502/components/title/title.component';
 import { FeaturedComponent } from './pages/collection-502/components/featured/featured.component';
-import { MatSlideToggleModule } from '@angular/material/slide-toggle';
+import { MatLegacySlideToggleModule as MatSlideToggleModule } from '@angular/material/legacy-slide-toggle';
```

Comment:
Also would be interested in this documentation as well

Reply:
documentation: https://material.angular.io/guide/mdc-migration
See my comment under src/app/admin/admin.module.ts for more explanation

Reply:
https://material.angular.io/guide/mdc-migration